src/main/java/com/amazon/ion/impl/bin/PresenceBitmap.kt

```diff
@@ -55,7 +55,7 @@ internal class PresenceBitmap {
         const val RESERVED = 0b11L
 
         private const val TWO_BIT_MASK = 0b11L
-        private const val PRESENCE_BITS_SIZE_THRESHOLD = 2
+        private const val PRESENCE_BITS_SIZE_THRESHOLD = 0
```

Comment:
With this change, a presence bitmap of at least one byte will be written whenever there exists at least 1 non-`!` parameter.